pages/transcripts/christoph-nakazawa.md

```diff
@@ -0,0 +1,187 @@
+
+----
+
+**Q:** Is it an eventual goal for react-test-renderer to provide functionality similiar to enzyme?
+
+**A:** react-test-render should absolutely have more features and we have an experimental version of it that we use at Facebook, but I cannot comment on the plans of the React team and don't want to make assumptions about their roadmap. However, we do need a way to control React Native components in tests, whether it is for snapshot tests or not. I recommend reaching out to Alex ( https://twitter.com/alex_frantic?lang=en ) and update you on the state of the react-test-renderer. He's been playing around with an extended version for a while.
+
+----
+
+**Q:** Earlier you had said that the tools you've built were to serve your own use cases, but a ton of these tools have been adopted by the community.  So, now you have bugs, feature requests, etc. flowing in from the outside and you have to spend time on these.  Do you feel like this is going to reach critical mass as the number of FB's open source tools grows?  Do you have a strategy for managing this?  A good example is Relay, as Relay is a beast and hasn't gotten big traction yet.
+
+**A:** Oh yeah, man, that's absolutely a problem. Especially with people that feel entitled that the open source project you are giving away for free has to be maintained day and night and that you are responsible for fixing issues they run into, not them.
+
+For Jest, we've been running the project more strict recently: I tend to say "no" more often and shut people down for PRs or feature requests that shouldn't be part of the project. This is often tough and it feels as if the lines are drawn arbitrarily. For projects like Yarn that are super active, we have started to do "bug triage" sessions, which worked really well to reduce the number of issues.
+
+Generally though, even if there is a large number of issues, there is nothing that forces us as maintainers to look at them and quite often we just don't have time to look at everything, even if we want to. I wish there was a way to split GitHub's issue tracker into an approved list of issues vs. a list of issues that come in. Finally, what works the best, is to find ways to align incentives between the business needs and the open source needs. For example, if you need to build a specific API then it makes sense to skim over issues to see how many people could benefit from it and let that influence your design. Sometimes you can solve many use-cases with one very smart technical decision and it requires a lot of creativity to find those.
```

Comment:
Maybe change the last sentence: "Sometimes you can solve many use-cases with one very smart technical decision which often requires a lot of creativity."